### DISCUSSION TITLE
amazon forwarded email address -- what is it?

### DISCUSSION THREAD

**Discussion by rroller** [moralmunky/Home-Assistant-Mail-And-Packages#678] [Q&A] [answered]
Thanks for this integration. Can someone explain to me what "amazon forwarded email address" is?

I setup my gmail to forward to another gmail address that this integration uses. It's not clear to me what should be used for the "amazon forwarded email address"

**Comment by firstof9** [comment, marked as answer]
Normally you'd use this setting for inputting the email address that your are forwarding from, but since gmail basically forwards the email without modifying the from headers in the email you _shouldn't_ need to use it.

**Reply by rroller** [reply]
Got it. Thanks!
